### PR TITLE
updated rule about external links and pdfs opening in the same window

### PR DESCRIPTION
**Pull request by benguhin**
per discussion at #294

### CODE REVIEW

File: ui-toolkit/links.md [hunk 1]
```diff
@@ -159,7 +159,9 @@ Carets (minicon glyph EE02) can emphasize CFPB webpages, such as in a navigation
 
 <div class="content-50 content-first">
 ##### External links
-Use the external link (glyph E610) to emphasize a non-CFPB webpage. External links should open in a new window or tab.
+Use the external link (glyph E610) to emphasize a non-CFPB webpage.
```
Comment (Scotchester) [inline ui-toolkit/links.md]: This sentence is missing the word "icon" or "minicon".

File: ui-toolkit/links.md [hunk 2]
```diff
@@ -159,7 +159,9 @@ Carets (minicon glyph EE02) can emphasize CFPB webpages, such as in a navigation
...
+
+External links links should open in the same browser window (i.e., do not set them to ```target="_blank"```). This allows the user to choose whether they want to open an additional window in order to view the content.
```
Comment (Scotchester) [inline ui-toolkit/links.md]: Duplicate word "links".

**Comment by benguhin** [comment]
good catches @Scotchester - just made those changes (and yes, I realize that I included your enterprise handle in my comment)

**Comment by Scotchester** [comment]
Thanks! :+1:

**Pull request merged**